text/0118-search-api.md

```diff
@@ -18,14 +18,14 @@ Meilisearch exposes 2 routes to perform searches:
 
 - 🔴 If the index does not exist, the API returns an [index_not_found](0061-error-format-and-definitions.md#index_not_found) error.
 
-If the instance is secured by a master-key, the auth layer will return the following errors:
+If the instance is secured by a master-key, the auth layer returns the following errors:
```

Comment:
I would love to be able to continue to make the specifications modular! As it's markdown, I already started to put more and more cross-link ref to avoid re-explaining things, it could work for this kind of error (Auth related)!

Reply:
Oh okay, you mean the changes to the Search API spec?

Reply:
Yes! :D Since this spec talks about just the typo tolerance, and this change is related to search :)